index.bs

```diff
@@ -838,7 +838,7 @@ This section discusses the box requirements for an [=AVIF file=] containing only
 <h4 id="avif-required-boxes">Minimum set of boxes</h4>
 
 <p>As indicated in [[#file-constraints]], an [=AVIF file=] is a compliant [[!MIAF]] file. As a consequence, some [[!ISOBMFF]] or [[!HEIF]] boxes are required, as indicated in the following table. The order of the boxes is indicative in the table. The specifications listed in the "Specification" 
-column may require a specific order for the box or for its children and shall be respected. For example, per [[!ISOBMFF]], the <code>[=FileTypeBox=]</code> is required to appear first in an [=AVIF file=].
```

Comment:
This long sentence is of the form:

> The specifications [...] may require a specific order for the box [...] and shall be respected.

So it means (boldface added for emphasis):

> The specifications [...] may require a specific order for the box [...] and **the specifications** shall be respected.

I believe it is intended to mean:

> The specifications [...] may require a specific order for the box [...] and **the order** shall be respected.